app/views/course/assessment/answer/_guided.html.slim

```diff
@@ -0,0 +1,2 @@
+- if answer.attempting?
+  = base_answer_form.button :button, t('common.submit'), value: answer.id, name: 'submit_answer'
```

Comment:
@lowjoel Oh I thought you agreed to use button, I can switch to use hidden field, what do you think ?